python/pyarrow/parquet/core.py

```diff
@@ -767,13 +767,16 @@ def _sanitize_table(table, new_schema, flavor):
     Other features such as compression algorithms or the new serialized
     data page format must be enabled separately (see 'compression' and
     'data_page_version').
-use_dictionary : bool or list
+use_dictionary : bool or list, default True
     Specify if we should use dictionary encoding in general or only for
     some columns.
-compression : str or dict
+    When encoding the column, if the dictionary size is too large, the
+    column will fallback to fallback encoding. Specially, ``BOOLEAN`` type
```

Comment:
Is this fallback encoding always PLAIN?

Reply:
Yes. Even the fallback method is called `FallbackToPlainEncoding()`, but we may switch to other encodings in the future( arrow-rs and parquet-mr allow some different encoding in format-v2 )

Reply:
You can refer to https://github.com/apache/arrow/issues/15165

I change here to `PLAIN`, and if this issue is supported, we can change the wording here.